### PR TITLE
perf: add message extension query sample

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: query-org-user-with-message-extension-sso/bot/adaptiveCards/learn.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (MuyangAmigo) [inline query-org-user-with-message-extension-sso/bot/adaptiveCards/learn.json:1]: Do we need those in the sample?

File: query-org-user-with-message-extension-sso/bot/adaptiveCards/welcome.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (MuyangAmigo) [inline query-org-user-with-message-extension-sso/bot/adaptiveCards/welcome.json:1]: Also I guess we don't need this file as well?

File: query-org-user-with-message-extension-sso/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Query Org User with Message Extension SSO
+This is a simple search-based message extension app demonstrating how to integrate with Bot Framework SDK, TeamsFx SDK and Microsoft Graph API to implement a feature that queries organization's user with single sign on (SSO) capability.
```
Comment (MuyangAmigo) [inline query-org-user-with-message-extension-sso/README.md:2]: Can we insert a screenshot / gif of this app upfront?

File: query-org-user-with-message-extension-sso/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+1. Stop debugging in Visual Studio Code.
+2. Sign up an ngrok account in https://dashboard.ngrok.com/signup.
+3. Copy your personal ngrok authtoken from https://dashboard.ngrok.com/get-started/your-authtoken.
+4. Run `npx ngrok authtoken <your-personal-ngrok-authtoken>` in Visual Studio Code terminal.
+5. Start debugging the project again by hitting the F5 key in Visual Studio Code.
+
+## Further reading
+- [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
+- [Add Authentication to Message Extension](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/add-authentication)
+- [Message Extension SSO](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/enable-sso-auth-me)
```
Comment (MuyangAmigo) [inline query-org-user-with-message-extension-sso/README.md:72]: Let's add a code of conduct section. See other sample: https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/share-now

File: query-org-user-with-message-extension-sso/.gitignore [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+.backup
+.fx/configs/localSettings.json
+.fx/states/*
+.DS_Store
+.archive
+.archive.log
+.env.teamsfx.local
+subscriptionInfo.json
+.fx/configs/config.local.json
+.fx/states/state.local.json
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/.gitignore:12]: seems redundant with `.fx/states/*`

File: query-org-user-with-message-extension-sso/.gitignore [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+node_modules
+.backup
+.fx/configs/localSettings.json
+.fx/states/*
+.DS_Store
+.archive
+.archive.log
+.env.teamsfx.local
+subscriptionInfo.json
+.fx/configs/config.local.json
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/.gitignore:11]: config.local.json file should not be ignored

File: query-org-user-with-message-extension-sso/bot/package.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "echobot",
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/bot/package.json:2]: Update the name

File: query-org-user-with-message-extension-sso/bot/package.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  "version": "1.0.0",
+  "description": "Microsoft Teams Toolkit hello world Bot sample",
+  "author": "Microsoft",
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "scripts": {
+    "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
+    "dev": "nodemon --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./index.ts",
+    "build": "tsc --build && shx cp -r ./adaptiveCards ./lib/",
```
Comment (SLdragon) [inline query-org-user-with-message-extension-sso/bot/package.json:11]: Do we need copy public file as below?
`"build": "tsc --build && shx cp -r ./src/adaptiveCards ./lib/src && copyfiles src/public/*.html lib/",'

File: query-org-user-with-message-extension-sso/.gitignore (hunk 6 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/.gitignore:11]: thanks

File: query-org-user-with-message-extension-sso/README.md (hunk 3 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/README.md:2]: sure

File: query-org-user-with-message-extension-sso/README.md (hunk 4 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/README.md:72]: thanks, updated

File: query-org-user-with-message-extension-sso/bot/package.json (hunk 7 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/bot/package.json:2]: thanks

File: query-org-user-with-message-extension-sso/bot/package.json (hunk 8 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/bot/package.json:11]: thanks

File: query-org-user-with-message-extension-sso/bot/adaptiveCards/welcome.json (hunk 2 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/bot/adaptiveCards/welcome.json]: deleted, thanks

File: query-org-user-with-message-extension-sso/bot/adaptiveCards/learn.json (hunk 1 above)
Comment (wenytang-ms) [reply query-org-user-with-message-extension-sso/bot/adaptiveCards/learn.json]: deleted

**Review by SLdragon — APPROVED** [review]

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**